### PR TITLE
Files page with pepcus core.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**
Files page with pepcus core.

### CODE REVIEW

File: src/components/routeComponents/files/Files.jsx [hunk 1]
```diff
@@ -0,0 +1,850 @@
...
+        overflow-y: ${props => (props.isDisplayCondition ? 'unset' : 'scroll')};
+        position: fixed;
+        left: 0;
+        background-color: ${getThemeProps('palette.advancedSearch.color')};
+    `}
+    @media (max-width: 992px) and (orientation: landscape) {
+        position: relative;
+    }
+    ${({ theme }) => theme.media.down('md')`
+        max-height: ${props => (props.isDisplayCondition ? 'initial' : 'initial')};
```
Comment (Nikmahajan) [inline src/components/routeComponents/files/Files.jsx]: Unnecessary condition.

File: src/components/routeComponents/files/Files.jsx [hunk 2]
```diff
@@ -0,0 +1,850 @@
...
+        height: ${props => (props.isDisplayCondition ? '100%' : '100%')};
```
Comment (Nikmahajan) [inline src/components/routeComponents/files/Files.jsx]: same here

File: src/components/routeComponents/files/Files.jsx [hunk 3]
```diff
@@ -0,0 +1,850 @@
...
+        width: ${props => (props.isDisplayCondition ? '100%' : '260px')};
+        left: ${props => (props.isDisplayCondition ? 'auto' : 'none')};
+        margin: ${props => (props.isDisplayCondition ? '0' : '0')};
```
Comment (Nikmahajan) [inline src/components/routeComponents/files/Files.jsx]: ?

File: src/components/routeComponents/files/Files.jsx [hunk 4]
```diff
@@ -0,0 +1,850 @@
...
+        overflow-y: ${props => (props.isDisplayCondition ? 'inherit' : 'scroll')};
+        position: ${props => (props.isDisplayCondition ? 'relative' : 'relative')};
+        display: ${props => (props.isDisplayCondition ? 'block' : 'none')}
+    `}
+    ${({ theme }) => theme.media.down('sm')`
+        margin: ${props => (props.isDisplayCondition ? '0' : '0')};
+        display: ${props => (props.isDisplayCondition ? 'block' : 'none')}
+    `}
+`;
```
Comment (Nikmahajan) [inline src/components/routeComponents/files/Files.jsx]: These type of errors should be corrected on self reviewing stage. Try to review carefully yourself before creating a PR

File: src/components/routeComponents/files/Files.jsx [hunk 5]
```diff
@@ -0,0 +1,850 @@
...
+    width: 100%;
+    max-width: 78%
+    padding-top: 10px;
+    margin: 0 0 0 300px;
+    overflow-x: hidden;
+    ${({ theme }) => theme.media.down('lg')`
+        display:  ${props => (props.isDisplayMessage ? 'unset' : 'none')}
+        max-width: ${props => (props.isMobile ? '100%' : '64%')}
+        margin: ${props => (props.isMobile ? '0' : 'unset')};
+        file-Component-mobile-wrapper margin: ${props => (props.isMobile ? 'unset' : '0 0 0 270px')};
```
Comment (Nikmahajan) [inline src/components/routeComponents/files/Files.jsx]: Is it a style property or any custom?

File: src/components/routeComponents/files/Files.jsx [hunk 6]
```diff
@@ -0,0 +1,850 @@
...
+    line-height: 20px;
+    padding: 10px;
+    font-size: 14px;
+    border-radius: 3px;
+    cursor: pointer;
+    text-decoration: none;
+    &:hover {
+        transition: 0.3s all;
+        background-color: ${getThemeProps('typography.titleFieldColor.color')};
+}
```
Comment (Nikmahajan) [inline src/components/routeComponents/files/Files.jsx]: Indentation

File: src/components/routeComponents/files/Files.jsx (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/components/routeComponents/files/Files.jsx]: Done

File: src/components/routeComponents/files/Files.jsx (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/components/routeComponents/files/Files.jsx]: Done.

File: src/components/routeComponents/files/Files.jsx (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/components/routeComponents/files/Files.jsx]: Done

File: src/components/routeComponents/files/Files.jsx (hunk 4 above)
Comment (BhagyashreeBhoyar) [reply src/components/routeComponents/files/Files.jsx]: Done

File: src/components/routeComponents/files/Files.jsx (hunk 5 above)
Comment (BhagyashreeBhoyar) [reply src/components/routeComponents/files/Files.jsx]: Done.

File: src/components/routeComponents/files/Files.jsx (hunk 6 above)
Comment (BhagyashreeBhoyar) [reply src/components/routeComponents/files/Files.jsx]: Done.

**Pull request merged**